security.qmd

```diff
@@ -1,3 +1,34 @@
 # Security {{< iconify fa6-solid lock >}} {#sec-security}
 
-xxx
\ No newline at end of file
+## Package security
+
+Keeping secrets a secret is an important task and should not be an afterthought - but instead considered from the beginning of a package.
+
+In general, the rOpenSci Development Guide has good advice on [package security](https://devguide.ropensci.org/package-development-security-best-practices.html).
+
+As time goes on we'll document here if we have any strong opinions about security related aspects of packages.
+
+## Data
+
+Software created in the WILDS may touch PII/PHI data.
+
+### Package Data
+
+Do not include any PII/PHI data within a package. If data needs to be included in a package for tests, examples, etc., use fake data that is similar enough to be able to run tests/examples, etc.
+
+There's many packages in both R (e.g. [charlatan][], [wakefield][]. [fakir][]) and Python (e.g. [faker][]) for creating fake data.
+
+### Examples
+
+We strongly encourage all exported, user-facing functions, to have examples. However, when PII/PHI data is a crucial part of an example for a function/class/etc. we need to take extra precautions.
```

Comment:
I don't think that anything in the WILDS would have PII/PHI data. If data is [de-identified](https://sciwiki.fredhutch.org/datascience/deidentification/), it's not considered to be PHI. 

I am not sure how I am going to do data-as-a-product on the CARDS platform and if would be possible/advisable to create R packages that someone live just on there. I think that is the only time that we would have PHI in an R package though.

Reply:
Okay, thanks very much @monicagerber ! Do you think we just remove this comment about PII/PHI data in examples?

Reply:
Definitely nothing in WILDS should contain PII/PHI in the package itself, but I imagine several WILDS packages will touch/transmit PII/PHI.